Assignment1/Activity2.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tables>
+    <headings>
+        <heading>Empno</heading>
+        <heading>Empname</heading>
+        <heading>Salary</heading>
+    </headings>
```

Comment:
these should have been used as tag names for each person, not separate set of elements